example-conf.yml

```diff
@@ -566,6 +569,25 @@ relay:
     #
     #clientbot_modesync: none
 
+###
+#
+# Serv Protect
+#
+# If you use this please uncomment the following values
+# and set them to what you want them to be.
+#
+###
+
+#servprotect:
+    # THINK: LENGTH KILL/SAVE's in TIME seconds
+
+    # length: How many KILL/SAVE's before a disconnect
+    #length: 5
+
+    # time: How long till a period expires?
```

Comment:
This is ambiguous. Better wording might be "Determines how much time should pass before the servprotect cache is reset.", but I'm open to any better suggestions.